webpack.config.js

```diff
@@ -94,7 +94,14 @@ module.exports = (env = {}) => {
           test: /[\/\\]CahierExercices[\/\\].+\.md$/,
           use: [
             require.resolve("html-loader"),
-            require.resolve("markdown-loader"),
+            {
+              loader: require.resolve("markdown-loader"),
+              options: {
+                highlight: function (code, lang) {
+                  return Prism.highlight(code, Prism.languages[lang], lang);
```

Comment:
Where is `Prism` defined?

Reply:
No idea, I kind of copy paste from the highlight function in https://github.com/Zenika/sensei/blob/7b3926b62c716f9c2d3d316dd92ed1bc91f12fb8/src/loaders/revealjs-loader.js#L210 and it magically worked.
I guess that when putting all code together it end up by having a `Prism` already defined.

BTW, I should do a real copy/paste to fix the case when unknown lang.

Reply:
I will add a proper import and definition to be clean.

Reply:
I did add explicit import.
Maybe we could mutualize the highlight function and prism import in a specific file?